docs/layers/accounts/deploy-accounts.mdx

```diff
@@ -88,8 +84,11 @@ This step-by-step process outlines how to deploy AWS accounts using `atmos` work
 
     These deployments will create all AWS member accounts and store relevant account metadata as "mappings" in the Terraform
     outputs of the `account-map` component. Rather than querying this `account` component each time we need an Account ID or
-    role, we provision a static component `account-map`. **IMPORTANT:** Always run
-    `atmos terraform apply account-map -s core-gbl-root` after provisioning accounts.
+    role, we provision a static component `account-map`.
+
+    <Note title="IMPORTANT">
```

Comment:
I believe we use CSS to make it uppercase. 

```suggestion
    <Note title="Important">
```

Reply:
replaced